### PR TITLE
Friendships v2 [Status Fix]

### PR DESCRIPTION
**Pull request by oscardelalanza**

### CODE REVIEW

**Comment by oscardelalanza** [comment]
This line of code in views/users/show.html.erb
https://github.com/zhuchini/ror-social-scaffold/blob/240e2b85653b5ae7bd7a774d54ca6b68ce6e0282/app/views/users/show.html.erb#L9

was changed to the following

https://github.com/oscardelalanza/ror-social-scaffold/blob/5129b71c44e337d19dad54b38f0af24854025e71/app/views/users/show.html.erb#L9

**Comment by oscardelalanza** [comment]
In the firendships controller the following method
https://github.com/zhuchini/ror-social-scaffold/blob/240e2b85653b5ae7bd7a774d54ca6b68ce6e0282/app/controllers/friendships_controller.rb#L12-L24

was changed to the following
https://github.com/oscardelalanza/ror-social-scaffold/blob/5129b71c44e337d19dad54b38f0af24854025e71/app/controllers/friendships_controller.rb#L12-L23

when you create a new friend request from `User A` to `User B` the following will happen:

* `User A` creates the request, that means that this is the `current_user`
* `User B` is the requested friend
* In friendshipsController#create the following is executed  
    - `@friendship = current_user.friendships.build(friend_id: params[:friend_id])` with this linen you are working with a `User object` which is used to build a new `friendship` record.
    - This means, that `current_user` (`User A`) is being referenced in the `friendships table` as `user_id` because the `friendship model` was marked whit the relation `belongs_to :user`
    - Basically when you create a new `friend request` from `User A` which is the `current_user` to `User B`, the `current_user.id` is going to be stored as `user_id` and `User B.id` (which is the requested user) as `friend_id ` in the friendships table.

So, when you switch to a requested user (`user B`) and execute the `update` method to change the `status` to `true` in the `relationships` table, you need to find the request by the `user_id` which is the id from the user that previously creates the request, in this case `User A` and the rest is assing the new value and save the `@friendship` object
